tox.ini

```diff
@@ -23,27 +25,16 @@ deps =
     pinned: six==1.9.0
     pinned: werkzeug==0.9.6
 
-    latest: eventlet  # need something
+whitelist_externals = make
 
 commands =
-    pip install --editable .[dev]
-    flake8 nameko test
-    pylint --rcfile=pylintrc nameko -E
-    coverage run --concurrency=eventlet --source nameko -m \
-        pytest test {posargs:--timeout=30}
-    coverage report --show-missing --fail-under=100
+    codebase: pip install --editable .[dev]
```

Comment:
Ah, I wondered what happened to this! I saw the email but couldn't find it...

``[testenv:lib]`` won't be matched by anything. The name has to exactly match one of the expanded  values (e.g. ``py27-latest-lib``).